spec/factories/tasks.rb

```diff
@@ -5,5 +5,11 @@
     from = Date.parse("2019/08/01")
     to   = Date.parse("2019/12/31")
     deadline { Random.rand(from..to) }
+    association :project
+    
+    trait :done do
+      status {:done}
```

Comment:
半角スペースで区切った方が見やすいですね
```suggestion
      status { :done }
```

Reply:
d96d75e でインデントを修正しました。